src/Speckle.Objects/BuiltElements/Archicad/ArchicadBeam.cs

```diff
@@ -117,6 +116,7 @@ internal set
   public double? coverFillTransformationYAxisX { get; set; }
   public double? coverFillTransformationYAxisY { get; set; }
 
+  [SpeckleType("Objects.BuiltElements.Archicad.ArchicadBeam+BeamSegment")]
```

Comment:
good job spotting this nightmare of a speckle_type... 🤮
You've correctly handled it, but we should at somepoint enforce no nested `Base` classes.